src/helpers/waitUntilComplete.ts

```
@@ -0,0 +1,11 @@
+import { type LensClient, LensTransactionStatusType } from '@lens-protocol/client';
+
+export async function waitUntilComplete(client: LensClient, hash: string | null) {
+    if (!hash) return;
```

CC @nuanyang233

---

Should throw a error.